canary/producer-cpp/scripts/generate-iot-credentials.sh

```diff
@@ -58,10 +67,12 @@ cat > iam-permission-document.json <<EOF
             "Action": [
                 "kinesisvideo:DescribeStream",
                 "kinesisvideo:PutMedia",
-                "kinesisvideo:TagStream",
-                "kinesisvideo:GetDataEndpoint"
+                "kinesisvideo:GetDataEndpoint",
+                "kinesisvideo:CreateStream"
             ],
-            "Resource": "arn:aws:kinesisvideo:*:*:stream/\${credentials-iot:ThingName}/*"
```

Comment:
Dp we want to retain this as it restricts access to only certain resources?

Reply:
This is ok since this is for canaries.